nipype/interfaces/fsl/tests/test_Level1Design.py

```diff
@@ -0,0 +1,12 @@
+import os
+from ...base import Undefined
+from ..model import Level1Design
+
+def test_level1design():
+	l = Level1Design()
+	runinfo = dict(cond=[{'name': 'test_condition', 'onset': [0, 10], 'duration':[10, 10]}],regress=[])
+	runidx = 0
+	contrasts = Undefined
+	usetd = False
+	do_tempfilter = False
+	return Level1Design._create_ev_files(l,os.getcwd(),runinfo,runidx,usetd,contrasts,do_tempfilter,"hrf")
```

Comment:
this doesn't really test it. the following would run a test. `something` would need to be replaced with the expected string

```
output = Level1Design._create_ev_files(l,os.getcwd(),runinfo,runidx,usetd,contrasts,do_tempfilter,"none")
yield assert_true, 'something' in output
output = Level1Design._create_ev_files(l,os.getcwd(),runinfo,runidx,usetd,contrasts,do_tempfilter,"dgamma")
yield assert_true, 'something' in output
output = Level1Design._create_ev_files(l,os.getcwd(),runinfo,runidx,usetd,contrasts,do_tempfilter,"gamma")
yield assert_true, 'something' in output
```